core-feature-pack/common/src/main/resources/content/bin/jboss-cli.sh

```diff
@@ -74,6 +76,37 @@ else
     JAVA_OPTS="$JAVA_OPTS -Djboss.modules.system.pkgs=com.sun.java.swing"
 fi
 
+# Determine whether ascii coloring could be used
```

Comment:
@lvydra how did you determine that algorithm? Is is straight from David's description in the JIRA issue?

Reply:
Hi @jmesnil  No, this is quite different approach. I have done some research, and this one seems to be a little bit more suitable in this context.